src/test/java/htsjdk/tribble/index/IndexTest.java

```diff
@@ -92,6 +88,7 @@ public void testWriteIndex(final File inputFile, final IndexFactory.IndexType ty
         Assert.assertTrue(tempIndex.exists());
         // load the generated index
         final Index loadedIndex = IndexFactory.loadIndex(tempIndex.getAbsolutePath());
+        //TODO: This not very useful; the test can pass even if the generated index is unuseable for queries.
```

Comment:
I implemented this test only for checking if writing the index does not blow up, and if the in-memory index and the one written have the same contents. For testing if it is usable for queries, it should be a different test (which actually does not require to write the index down, I guess).